ambry-store/src/test/java/com.github.ambry.store/PersistentIndexTest.java

```diff
@@ -712,7 +712,8 @@ public void testMissingEntries()
   }
 
   @Test
-  public void testRollingIndex() {
+  public void testRollingIndex()
+      throws IOException {
     MockClusterMap map = null;
```

Comment:
Not a big deal, but it might be good to change these variable names from `map` to `mockClusterMap` for consistency with the hard delete test.